### DISCUSSION TITLE
Is there some way to allow firejail to work with symbolic links outside the home directory?

### DISCUSSION THREAD

**Discussion by diamond-lizard** [netblue30/firejail#5212] [Q&A] [answered]
I'm trying to run Firefox under firejail like so:

    firejail --private --private-home=.mozilla firefox -no-remote

But it errors out with:

    Error: .asoundrc is a symbolic link pointing to a file outside home directory
    Error: proc 3765 cannot sync with peer: unexpected EOF
    Peer 3768 unexpectedly exited with status 1

My ```~/.asoundrc``` is symlinked to another directory with [GNU Stow](https://www.gnu.org/software/stow/), which firejail [doesn't seem to like](https://github.com/netblue30/firejail/discussions/4529).

I'm wondering if there's some way around this, or if firejail is just incompatible with this use of GNU Stow?

**Comment by rusty-snake** [comment, marked as answer]
> Is there some way to allow firejail to work with symbolic links outside the home directory?

No.
